var/spack/repos/builtin/packages/libproxy/package.py

```diff
@@ -26,11 +26,18 @@ class Libproxy(CMakePackage):
     depends_on("perl", type=("build", "run"), when="+perl")
     depends_on("python", type=("build", "run"), when="+python")
 
+    extends("python", when="+python")
+
     def cmake_args(self):
-        return [
+        args = [
             self.define_from_variant("WITH_PERL", "perl"),
             self.define_from_variant("WITH_PYTHON3", "python"),
             self.define("WITH_DOTNET", False),
             self.define("WITH_PYTHON2", False),
             self.define("WITH_VALA", False),
         ]
+        if "+python" in self.spec:
+            pynver = "python{0}".format(self.spec["python"].version.up_to(2))
+            pysite = join_path(self.prefix.lib, pynver, "site-packages")
+            args.append(self.define("PYTHON3_SITEPKG_DIR", pysite))
```

Comment:
```suggestion
            args.append(self.define("PYTHON3_SITEPKG_DIR", python_platlib))
```
The exact directory changes depending on OS, Python version, external vs. internal, time of day, mood of your computer, etc, etc. Let me know if this doesn't work, sometimes it's finicky and thinks `python_platlib` isn't defined.